private/extensions/maven.bzl

```diff
@@ -344,6 +346,13 @@ def maven_impl(mctx):
 
             repos[install.name] = repo
 
+    for (repo_name, known_names) in repo_name_2_module_name.items():
+        if len(known_names) > 1:
+            print("The maven repository '%s' has contributions from multiple bzlmod modules, and will be resolved together: %s" % (
+                repo_name, # e.g. "maven"
+                str(known_names), # e.g. bzl_module_foo, bzl_module_bar
```

Comment:
Can we sort the list of `known_names` before displaying?

Reply:
Or keep them in order as they were iterated by `mctx.modules`?